lib/SILOptimizer/Mandatory/PredictableMemOpt.cpp

```diff
@@ -233,7 +234,9 @@ struct AvailableValue {
     InsertionPoints.set_union(Other.InsertionPoints);
   }
 
-  void addInsertionPoint(StoreInst *I) & { InsertionPoints.insert(I); }
+  void addInsertionPoint(SILInstruction *I) & {
```

Comment:
Did you do this because you have a follow up PR that adds different types of insertion points? By itself this change doesn't make sense. It's better to raise the assumed type up to the interface.

Reply:
Sure. I realized I did this for a stupid reason.